### PR TITLE
Profile page UI

### PR DESCRIPTION
**Pull request by anamika7153**
### Closes: #601 

|   Before  |   Updated                |
|-----|---------------------------------|
| image |  |
| image |  |
| image |  |

### CODE REVIEW

File: src/pages/Profile/index.css [hunk 1]
```diff
@@ -24,9 +24,79 @@
...
+@media screen and (max-width: 1200px) {
+  .profile-section {
+    width: calc(100% - 15vw) !important;
+  }
+  .back-btn {
+    margin-left: 5rem !important;
+  }
+}
+
+@media screen and (max-width: 800px) {
```
Comment (narayan954) [inline src/pages/Profile/index.css:94]: why are there too many media queries for even similar sizes?

**Review by narayan954 — CHANGES_REQUESTED** [review]
can we please reduce/optimise media queries?
also, the back button is lagging as you can see
image

Also mobile display is like this now
image

**Comment by anamika7153** [comment]
@narayan954 I'm on it.

**Comment by narayan954** [comment]
> @narayan954 I'm on it.

best of luck!

**Comment by narayan954** [comment]
Also, maybe we dont need that back button here, as other options are available directly in the sidebar. You may remove that

**Comment by anamika7153** [comment]
> Also, maybe we dont need that back button here, as other options are available directly in the sidebar. You may remove that

Noted

**Comment by anamika7153** [comment]
### Screenshots

| | |
|---|-----|
|image | image |
| image |image |

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
It looks good to me!

**Pull request merged**